store/auth.go

```diff
@@ -225,3 +227,39 @@ func RefreshDischargeMacaroon(discharge string) (string, error) {
 
 	return requestDischargeMacaroon(UbuntuoneRefreshDischargeAPI, data)
 }
+
+// RequestStoreDeviceNonce requests a nonce for device authentication against the store.
+func RequestStoreDeviceNonce() (string, error) {
```

Comment:
Is this really about a device?  The URL seems more generic than that.

Also, I don't quite understand yet what's the rationale we use for deciding when methods should go into *Store and when they shouldn't. Is there a clear line?